scripts/add_existing_baseyear.py

```diff
@@ -24,6 +24,10 @@
 from add_electricity import sanitize_carriers
 from prepare_sector_network import cluster_heat_buses, define_spatial, prepare_costs
 
+from scripts.definitions.heat_sector import HeatSector
+from scripts.definitions.heat_system import HeatSystem
+from scripts.definitions.heat_system_type import HeatSystemType
```

Comment:
small thing, but I think one could just `from definitions.heat_sector import HeatSector`

Reply:
Thanks for spotting. Happy to change it in #1180